presto-docs/src/main/sphinx/release/release-338.rst

```diff
@@ -0,0 +1,68 @@
+=========================
+Release 338 (XX Jul 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect results when joining tables on a masked column. (:issue:`4251`)
+* Fix planning failure when multiple columns have a mask. (:issue:`4322`)
+* Fix incorrect comparison for ``TIMESTAMP WITH TIME ZONE`` values with precision larger than 3. (:issue:`4305`)
+* Fix incorrect rounding for timestamps before 1970-01-01. (:issue:`4370`)
+* Fix query failure when using ``VALUES`` with a ``real`` or ``double`` ``NaN`` value. (:issue:`4119`)
+* Fix query failure when joining tables on a ``real`` or ``double`` column and one of the joined tables
+  contains ``NaN`` value. (:issue:`4272`)
+* Fix incorrect query results when Parquet file has no min/max statistics for an integral column. (:issue:`4200`)
+* Fix unauthorized error for internal requests to management endpoints. (:issue:`4304`)
+* Fix memory leak while using dynamic filtering. (:issue:`4228`)
+* Improve dynamic partition pruning for broadcast joins. (:issue:`4262`)
+* Add support for setting column comments via the ``COMMENT ON COLUMN`` syntax. (:issue:`2516`)
+* Add compatibility mode for non-compliant clients when rendering datetime type names with default precision
```

Comment:
How about "legacy" here rather than non-compliant

Reply:
I wanted to make emphasis on the fact that they shouldn't have hardcoded those assumptions, but I'll update as suggested.